samples/Dantooine/Dantooine.Client/Server/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace Dantooine.BFF.Server
+{
+    public class Program
```

Comment:
Since you used minimal APIs for the resource server, I think it would be nice to also them for the other projects that are part of the Dantooine sample.

Reply:
I don't think this is a good idea to switch to minimal APIs here